src/components/Item.js

```diff
@@ -1,14 +1,36 @@
 import React from "react";
 
-function Item({ item }) {
+function Item({ item, onUpdateItem, onDeleteItem }) {
+
+    function handleAddToCart() {
```

Comment:
You do this in almost every lab!! PLEASE PLEASE PLEASE PLEASE PLEASE try to get out of this habit. I'm going to make you go back and delete these lines until you you stop... sorry!
```suggestion
function Item({ item, onUpdateItem, onDeleteItem }) {
    function handleAddToCart() {
```